dali/operators/image/convolution/laplacian_params.h

```diff
@@ -131,7 +131,7 @@ class LaplacianWindows {
 template <int axes>
 std::array<float, axes> GetNormalizationFactors(
     const std::array<std::array<int, axes>, axes> &window_sizes) {
-  std::array<float, axes> factors;
+  std::array<float, axes> factors = {};
```

Comment:
This is sad. I consider it a change for the worse. Now, if our initialization logic is indeed broken, a smarter tool won't be able to tell us wrong, because the array would be initialized. I'd actually report it as false positive and tell coverity to fix their s**t.

Reply:
I wasn't sure if I should ignore those, but decided not to ignore them thinking that: if there is indeed a value that is not written with meaningful data one day, it at least will behave the same from run to run, so may be easier to catch?

Reply:
Though I mean it more in the context of class members rather than this one which is obviously over-cautious.

Reply:
Well... at least with uninitialized data we can use sanitizers, like valgrind.
```
#include <iostream>
using namespace std;

template <int n, int m>
struct Mat {
    Mat(float constant = 0.0f) {
        for (int i = 0; i < n; i++)
            for (int j = 0; j < n; j++)
                data[i][j] = constant;
    }
    float data[n][m] = {};
};

int main() {
    Mat<2, 3> m = 42;

    for (int i = 0; i < 2; i++) {
        for (int j = 0; j < 3; j++)
            cout << m.data[i][j] << " ";
        cout << "\n";
    }

    return 0;
}
```
According to valgrind, the above program is correct. Removing the initializer `= {}` makes it clear that it isn't.

Reply:
That's a good point. Okay, so let me revert those changes and ignore them in the Coverity.